pg_search/tests/pg_regress/expected/html_strip_filter.out

```diff
@@ -0,0 +1,132 @@
+-- Disable parallel workers to avoid differences in plans
+SET max_parallel_workers_per_gather = 0;
+SET enable_indexscan to OFF;
+SET paradedb.enable_mixed_fast_field_exec = true;
+SELECT 'The <b>test</b> of the <i>html_strip</i> filter'::pdb.unicode_words('html_strip=true')::text[];
+                text                 
+-------------------------------------
+ {the,test,of,the,html_strip,filter}
+(1 row)
+
+SELECT 'Tom&apos;s &quot;html_strip&quot; filter handles ampersands like A&amp;B correctly.'::pdb.unicode_words('html_strip=true')::text[];
+                              text                               
+-----------------------------------------------------------------
+ {tom's,html_strip,filter,handles,ampersands,like,a,b,correctly}
+(1 row)
+
+SELECT 'Invalid entities like &nosuchentity; &#xRT; &#-1; s == “&amp hej och hå”`'::pdb.unicode_words('html_strip=true')::text[];
+                            text                             
+-------------------------------------------------------------
+ {invalid,entities,like,nosuchentity,xrt,1,s,amp,hej,och,hå}
+(1 row)
+
+SELECT 'The <b>test</b> of the <i>html_strip</i> filter'::pdb.unicode_words('html_strip=false')::text[];
+                    text                     
+---------------------------------------------
+ {the,b,test,b,of,the,i,html_strip,i,filter}
+(1 row)
+
+SELECT 'Tom&apos;s &quot;html_strip&quot; filter handles ampersands like A&amp;B correctly.'::pdb.unicode_words('html_strip=false')::text[];
+                                        text                                        
+------------------------------------------------------------------------------------
+ {tom,apos,s,quot,html_strip,quot,filter,handles,ampersands,like,a,amp,b,correctly}
+(1 row)
+
+SELECT 'The <b>test</b> of the <i>html_strip</i> filter'::pdb.unicode_words::text[];
+                    text                     
+---------------------------------------------
+ {the,b,test,b,of,the,i,html_strip,i,filter}
+(1 row)
+
+SELECT 'Tom&apos;s &quot;html_strip&quot; filter handles ampersands like A&amp;B correctly.'::pdb.unicode_words::text[];
+                                        text                                        
+------------------------------------------------------------------------------------
+ {tom,apos,s,quot,html_strip,quot,filter,handles,ampersands,like,a,amp,b,correctly}
+(1 row)
+
+SELECT 'The <b>test</b> of the <i>html_strip</i> filter'::pdb.unicode_words('html_strip=true')::text[];
+                text                 
+-------------------------------------
+ {the,test,of,the,html_strip,filter}
+(1 row)
+
+SELECT 'Tom&apos;s &quot;html_strip&quot; filter handles ampersands like A&amp;B correctly.'::pdb.unicode_words('html_strip=true')::text[];
+                              text                               
+-----------------------------------------------------------------
+ {tom's,html_strip,filter,handles,ampersands,like,a,b,correctly}
```

Comment:
Unrelated, but: apostrophes should maybe be stripped in the unicode words filter.